src/handler/ts6.rs

```diff
@@ -87,7 +87,7 @@ impl TS6Handler {
         Outcome::Empty
     }
 
-    fn handle_line_sid(network: &mut Network, sid: &str, line: &Line) -> Outcome {
+    fn handle_line_sid(network: &mut Network, src_sid: &str, line: &Line) -> Outcome {
```

Comment:
given we now `#![allow(clippy::similar_names)]`, do we need to change these?

Reply:
"Need" no, but the `sid` and `uid` bindings further down in those `handle_line` functions generated warnings due to `#![warn(clippy::shadow_similar)]`.

Reply:
i wonder if this is solved to a degree by #9

Reply:
oh, now i see what "shadow" means